base/math.jl

```diff
@@ -277,7 +277,8 @@ asinh(x::Number)
 Accurately compute ``e^x-1``.
 """
 expm1(x)
-for f in (:exp2, :expm1)
+
+for f in (:exp2,)
```

Comment:
Now that this is down to a single function, it would make sense to get rid of the loop.

Reply:
Yes, I should have made a comment after I did this. The only reason why I did it like this is because it was easier for me, and the remaining code will be removed when I rebase https://github.com/JuliaLang/julia/pull/24859 after this is merged. Both should be quite close to merge-able I think.

I can remove the loop in this PR if you prefer it for a "clean" git history.

(on a side note, having removed these functions from the tuple one by one, I'm very happy to see the loop gone soon :))